workflows/eager.nf

```diff
@@ -352,6 +360,23 @@ workflow EAGER {
     }
 
     //
+    // MODULE: QUALIMAP_BAMQC
+    //
+
+    if ( !params.skip_qualimap ) {
+
+        qualimap_input = ch_dedupped_bams
+        .map {
+            meta, bam, bai ->
+            [ meta, bam ]
+        }
+
```

Comment:
Add a meta to the snpcapture_bed file and combine snpcapture bed and sample by reference id (see contamination with angsd for an example)

Reply:
subworkflows/local/estimate_contamination.nf lines 19-44